scripts/lay-down-os

```diff
@@ -115,39 +109,30 @@ EOF
 fi
 }
 
-format_and_mount()
+mount_fs()
 {
     device_defined ${DEVICE}
-    format_device
     mount_device
     trap "umount ${BASE_DIR}" EXIT
     create_boot_dirs
 }
 
-iso_install()
-{
-    format_and_mount
-    install_grub
-    grub2_config
-}
-
 if [ -n ${ENV} ]; then
     case ${ENV} in
-        "virtualbox-iso")
-            iso_install
+        "generic")
+            mount_fs
+            install_grub
+            grub2_config
```

Comment:
Just seemed more in line with your other case statements to just put the functions inline.